src/Microsoft.ML.AutoML/API/AutoCatalog.cs

```diff
@@ -538,55 +541,158 @@ public AutoMLExperiment CreateExperiment()
         /// <param name="inputColumnName">input column name.</param>
         internal SweepableEstimator[] TextFeaturizer(string outputColumnName, string inputColumnName)
         {
-            throw new NotImplementedException();
+            var option = new FeaturizeTextOption
+            {
+                InputColumnName = inputColumnName,
+                OutputColumnName = outputColumnName,
+            };
+
+            return new[] { SweepableEstimatorFactory.CreateFeaturizeText(option) };
         }
 
         /// <summary>
         /// Create a list of <see cref="SweepableEstimator"/> for featurizing numeric columns.
         /// </summary>
-        /// <param name="outputColumnName">output column name.</param>
-        /// <param name="inputColumnName">input column name.</param>
-        internal SweepableEstimator[] NumericFeaturizer(string outputColumnName, string inputColumnName)
+        /// <param name="outputColumnNames">output column names.</param>
+        /// <param name="inputColumnNames">input column names.</param>
+        internal SweepableEstimator[] NumericFeaturizer(string[] outputColumnNames, string[] inputColumnNames)
         {
-            throw new NotImplementedException();
+            Contracts.Check(outputColumnNames.Count() == inputColumnNames.Count() && outputColumnNames.Count() > 0, "outputColumnNames and inputColumnNames must have the same length and greater than 0");
```

Comment:
I think this will blow up if either of those inputs are null. Make sure they aren't null first.